docs/usage/configuration/config_documentation.md

```diff
@@ -663,6 +663,8 @@ This setting has the following sub-options:
    has missed. Disabled by default.
 * `notif_for_new_users`: Set to false to disable automatic subscription to email
    notifications for new users. Enabled by default.
+* `delay_before_mail`: Time we always wait before ever emailing about a notification
+   (to give the user a chance to respond to other push or notice the window). Defaults to 10 minutes.
```

Comment:
```suggestion
* `delay_before_mail`: The time to wait before emailing about a notification.
  This gives the user a chance to view the message via push or an open client.
  Defaults to 10 minutes.

   *New in Synapse 1.98.0._
```

Should this be `notif_delay_before_mail`? Some of other settings aren't related to notifications.